### ISSUE TITLE
Enable pre and post text of cite role

### ISSUE THREAD

**Issue by julian-belina** [executablebooks/mystmd#718]
Hello,

it would be great if the pre and post text feature of the cite role could be integrated into mystmd. This could be used integrate page numbers into your citations.

```
The axioms were introduced by:cite:t:`{see}1977:nelson`.
The axioms were introduced by:cite:t:`1977:nelson{p.1166}`.
The axioms were introduced by:cite:t:`{see}1977:nelson{p.1166}`.
Axioms were introduced :cite:p:`{see}1977:nelson`.
Axioms were introduced :cite:p:`1977:nelson{p.1166}`.
Axioms were introduced :cite:p:`{see}1977:nelson{p.1166}`.
```

https://readthedocs.org/projects/sphinxcontrib-bibtex/downloads/pdf/latest/ on page. 9

**Comment by rowanc1** [comment]
Thanks @julian-belina -- I have put together a PR to include this in #723. We should now support the prefix and suffix information using this syntax!

**Comment by rowanc1** [comment]
We should be releasing this later today! Thanks again for the improvement, looking forward to your next improvement/bug report!

**Comment by rowanc1** [comment]
This has been released in v1.1.25 🚀
